templates/settings.py.erb

```diff
@@ -1,3 +1,16 @@
+################################################################################
+# !!! WARNING: DO NOT EDIT THIS FILE !!!
+################################################################################
```

Comment:
I'd drop this line. The "file managed by Puppet" is actually the explanation for the warning.
```suggestion
```